spec/system/claims/support/create_a_claim_window_spec.rb

```diff
@@ -54,14 +54,16 @@ def then_i_see_the_form_to_create_a_claim_window
     expect(page).to have_css("legend", text: "Window closes")
   end
 
-  def when_i_fill_in_the_form(starts_on:, ends_on:)
-    fill_in "claims_claim_window[starts_on(3i)]", with: starts_on.day
-    fill_in "claims_claim_window[starts_on(2i)]", with: starts_on.month
-    fill_in "claims_claim_window[starts_on(1i)]", with: starts_on.year
+  def when_i_fill_in_the_form(starts_on:, ends_on:, academic_year:)
+    fill_in "claims_claim_window_form[starts_on(3i)]", with: starts_on.day
+    fill_in "claims_claim_window_form[starts_on(2i)]", with: starts_on.month
+    fill_in "claims_claim_window_form[starts_on(1i)]", with: starts_on.year
```

Comment:
I think it would be cleaner to specify labels rather than HTML IDs in these specs. Ideally system specs should interact with the UI as if they were a user – e.g. looking for buttons and labels, rather than the underlying HTML IDs of attributes.

```suggestion
    within_fieldset("Starts on") do
      fill_in "Day", with: starts_on.day
      fill_in "Month", with: starts_on.month
      fill_in "Year", with: starts_on.year
    end
```

We went through a similar wave over in Placements where we changed IDs to text labels because we found it made for more readable specs – for example, see https://github.com/DFE-Digital/itt-mentor-services/pull/856#discussion_r1684554702.